### PR TITLE
Distributed tracing default

### PR DESCRIPTION
**Pull request by priyanshi-gupta**
DO-NOT-MERGE

### What does this PR do?
1.APM rebranded as APM & Distributed Tracing on home page and left pane menu. Distributed Tracing enabled by default. Documented on the home page. Removed from advanced instrumentation section.
2. Opentracing for DT in C++ moved to OT section in advanced instrumentation
3. Added documentation link for Aiohttp in python, removed “experimental” word in ruby.
3. DT for C++ and  Go moved to manual instrumentation.

### CODE REVIEW

File: content/en/tracing/languages/python.md [hunk 1]
```diff
@@ -91,13 +95,13 @@ The `ddtrace` library includes support for the following libraries:
...
-| [Kombu][28]                     | Fully Supported | http://pypi.datadoghq.com/trace/docs/other_integrations.html#kombu    |
-| [Requests][29]               | Fully Supported | http://pypi.datadoghq.com/trace/docs/other_integrations.html#requests |
+| [Boto][26]                        | Fully Supported | http://pypi.datadoghq.com/trace/docs/other_integrations.html#boto     |
+| [Futures][27] | Fully Supported | http://pypi.datadoghq.com/trace/docs/other_integrations.html#futures  |
+| [Celery][28]                              | Fully Supported | http://pypi.datadoghq.com/trace/docs/other_integrations.html#celery   |
+| [httplib][29]            | Fully Supported | http://pypi.datadoghq.com/trace/docs/other_integrations.html#httplib  |
+| [Jinja2][30]                                    | Fully Supported | http://pypi.datadoghq.com/trace/docs/other_integrations.html#jinja2   |
+| [Kombu][31]                     | Fully Supported | http://pypi.datadoghq.com/trace/docs/other_integrations.html#kombu    |
+| [Requests][32]               | Fully Supported | http://pypi.datadoghq.com/trace/docs/other_integrations.html#requests |
```
Comment (priyanshi-gupta) [inline content/en/tracing/languages/python.md]: Remove requests as its already in the list

File: content/en/tracing/languages/python.md [hunk 2]
```diff
@@ -108,28 +112,31 @@ The `ddtrace` library includes support for the following libraries:
...
+[23]: https://redis.io
+[24]: https://www.sqlalchemy.org
+[25]: https://www.sqlite.org
+[26]: http://docs.pythonboto.org/en/latest
+[27]: https://docs.python.org/3/library/concurrent.futures.html
+[28]: http://www.celeryproject.org
+[29]: https://docs.python.org/2/library/httplib.html
+[30]: http://jinja.pocoo.org
+[31]: https://kombu.readthedocs.io/en/latest
+[32]: http://docs.python-requests.org/en/master
```
Comment (priyanshi-gupta) [inline content/en/tracing/languages/python.md]: remove requests link as its already in link line 14

**Comment by l0k0ms** [comment]
This is going to be heavily conflicting with https://github.com/DataDog/documentation/pull/4290, best move would be to check out from #4290 and start working from there

**Comment by l0k0ms** [comment]
Closing this PR as there are too many conflicts.

**Pull request closed without merging**